joltify/assetlist.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "joltify",
+  "assets": [
+    {
+      "description": "The native staking and governance token of Joltify",
+      "extended_description": "Joltify is the First EVM compatible game-changing L1 Public Chain built on the Cosmos SDK for Real-World Assets (RWA) and Beyond.",
+      "denom_units": [
+        {
+          "denom": "ujolt",
+          "exponent": 0
+        },
+        {
+          "denom": "jolt",
+          "exponent": 6
+        }
+      ],
+      "base": "ujolt",
+      "name": "Joltify",
+      "display": "jolt",
+      "symbol": "JOLT",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/joltify/images/jolt.svg"
```

Comment:
This SVG is a rather low quality PNG -> SVG conversion. I'd ask that we consider removing it